merlin/systems/dag/ops/fil.py

```diff
@@ -0,0 +1,91 @@
+import pathlib
+
+import numpy as np
+from google.protobuf import text_format  # noqa
+
+import merlin.systems.triton.model_config_pb2 as model_config  # noqa
+from merlin.dag import ColumnSelector  # noqa
+from merlin.schema import ColumnSchema, Schema  # noqa
+from merlin.systems.dag.ops.operator import InferenceOperator
+
+
+class FILPredict(InferenceOperator):
+    """Operator for Forest Inference Library (FIL) models.
+
+    Packages up XGBoost models to run on Triton inference server using the fil backend.
+    """
+
+    def __init__(self, model):
+        """Instantiate a FILPredict inference operator.
+
+        Parameters
+        ----------
+        model : xgboost.Booster
+            An XGBoost model.
+        """
+        self.model = model
+        self.input_schema = Schema([ColumnSchema("input__0", dtype=np.int32)])
```

Comment:
Since we have methods to compute the schemas, might make sense to return these schemas from those methods (even though they're functionally constants in this case.)

Reply:
that is a bit cleaner, updated